CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+List of incompatible ABI changes in this release:
+
+* The headerFormat member was added to avifEncoder.
```

Comment:
Delete this. This is a compatible change because `headerFormat` was added to the end of the `avifEncoder` struct.

Reply:
I thought so then found similar entries in 1.0.0, but forgot to check whether these were at the end of structs or not.

Reply:
We didn't declare a stable ABI until 1.0.0, so new members were added to the most logical places in the structs. After 1.0.0, we can add new members to the end of the `avifImage`, `avifDecoder`, and `avifEncoder` structs without breaking backward compatibility.